modules/ROOT/pages/secure-storage.adoc

```diff
@@ -0,0 +1,110 @@
+= Secure Storage Overview
+
+Anypoint Studio 7.10 and earlier versions stored the information of authenticated users in the workspace, meaning that you could have different authenticated users per workspace.
+
+Studio 7.11 introduces a Secure Storage facility provided by Eclipse to encrypt and save your credentials.
+
+Studio provides a master password provider that automatically generates a master password for all your user credentials. When adding a new credential, Studio uses your master password to encrypt and save that credential into a secure storage file in your file system.
+
+Because of the architecture of the secure storage, the authenticated users are now shared across all Eclipse-based workspaces and Anypoint Studio instances.
```

Comment:
We don't want to make this a "Studio" feature, because we  don't have control over it. We inherit this from Eclipse, and the user has no choice in whether using this or not.  

At some point I wanted to start the page with something like this:

> Because Anypoint Studio is an Eclipse-based application, it relies on the underlying existing architecture of the Eclipse platform. Therefore, Studio re-uses the Eclipse secure storage provided mechanism.

It might be necessary to convey the idea that this is how Studio works now and it should not bother the user that much.  Still, because it's a behavior change, it has to be documented. What do you think @kbostrom ?